src/java.base/share/classes/java/nio/Direct-X-Buffer.java.template

```diff
@@ -74,6 +66,8 @@ class Direct$Type$Buffer$RW$$BO$
     }
 
 #if[byte]
+    // Cached unaligned-access capability
+    protected static final boolean UNALIGNED = Bits.unaligned();
```

Comment:
Can you drop the protected modifier and check that it builds?

Reply:
Yes, it builds when I drop modifier.
```
    // Cached unaligned-access capability
    static final boolean UNALIGNED = Bits.unaligned();
```

Reply:
Indeed, we should remove the protected modifier in a package-private class.